### DISCUSSION TITLE
It look bad

### DISCUSSION THREAD

**Discussion by luckfunc** [community/community#63038] [Mobile] [answered]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, 版本 1.124.0 (105848740)
Device: iPhone 11 (iOS 15.3)
</details![image](https://github.com/community/community/assets/85545304/3dab2a64-1e55-4917-9608-aabc3233841f)

**Comment by mattruggio** [comment, marked as answer]
Thank you for the feedback @Ddkkkk11 but unfortunately I will need more specific information in order to help out. Please let us know if any specific concerns regarding the user interface/experience comes to mind in the future.
